These targets are dead code and they aren't available with system Abseil. Based on:
https://build.opensuse.org/package/view_file/openSUSE:Factory:ARM/nodejs-electron

--- third_party/abseil-cpp/BUILD.gn	2025-10-24 18:42:30.000000000 +0200
+++ third_party/abseil-cpp/BUILD.gn	2025-10-31 10:32:27.540854353 +0100
@@ -67,8 +67,6 @@
     "//third_party/abseil-cpp/absl/base",
     "//third_party/abseil-cpp/absl/base:config",
     "//third_party/abseil-cpp/absl/base:core_headers",
-    "//third_party/abseil-cpp/absl/base:fast_type_id",
-    "//third_party/abseil-cpp/absl/base:log_severity",
     "//third_party/abseil-cpp/absl/base:no_destructor",
     "//third_party/abseil-cpp/absl/base:nullability",
     "//third_party/abseil-cpp/absl/base:prefetch",
@@ -90,13 +88,9 @@
     "//third_party/abseil-cpp/absl/hash",
     "//third_party/abseil-cpp/absl/log:absl_check",
     "//third_party/abseil-cpp/absl/log:absl_log",
-    "//third_party/abseil-cpp/absl/log:absl_vlog_is_on",
     "//third_party/abseil-cpp/absl/log:die_if_null",
     "//third_party/abseil-cpp/absl/log:globals",
     "//third_party/abseil-cpp/absl/log:initialize",
-    "//third_party/abseil-cpp/absl/log:log_entry",
-    "//third_party/abseil-cpp/absl/log:log_sink",
-    "//third_party/abseil-cpp/absl/log:log_sink_registry",
     "//third_party/abseil-cpp/absl/memory",
     "//third_party/abseil-cpp/absl/meta:type_traits",
     "//third_party/abseil-cpp/absl/numeric:bits",
@@ -107,7 +101,6 @@
     "//third_party/abseil-cpp/absl/status",
     "//third_party/abseil-cpp/absl/status:statusor",
     "//third_party/abseil-cpp/absl/strings",
-    "//third_party/abseil-cpp/absl/strings:charset",
     "//third_party/abseil-cpp/absl/strings:cord",
     "//third_party/abseil-cpp/absl/strings:has_ostream_operator",
     "//third_party/abseil-cpp/absl/strings:str_format",
@@ -123,14 +116,11 @@
   # Dependencies that preferably shouldn't be public in chromium.
   public_deps += [
     # public in abseil, but deprecated in chromium, yet used.
-    "//third_party/abseil-cpp/absl/base:dynamic_annotations",
 
     # base/logging.cc uses this non-public absl dependency while there no better
     # alternative to inject abort hook. See notes in `base/logging.cc`.
-    "//third_party/abseil-cpp/absl/base:raw_logging_internal",
 
     # absl internal utilities used by protobuf.
-    "//third_party/abseil-cpp/absl/container:layout",
   ]
 
   if (is_component_build) {
@@ -178,7 +168,6 @@
     # the flag registration mechanism is problematic in component builds.
     "//third_party/abseil-cpp/absl/flags:config",
     "//third_party/abseil-cpp/absl/flags:flag",
-    "//third_party/abseil-cpp/absl/flags:marshalling",
     "//third_party/abseil-cpp/absl/flags:parse",
     "//third_party/abseil-cpp/absl/flags:reflection",
     "//third_party/abseil-cpp/absl/flags:usage",
@@ -256,20 +245,10 @@
       deps = [
         "absl/algorithm:algorithm_test",
         "absl/algorithm:container_test",
-        "absl/base:attributes_test",
         "absl/base:config_test",
-        "absl/base:iterator_traits_test",
         "absl/base:no_destructor_test",
-        "absl/base:nullability_default_nonnull_test",
         "absl/base:nullability_test",
-        "absl/base:nullability_traits_test",
-        "absl/base:poison_test",
         "absl/base:prefetch_test",
-        "absl/base:raw_logging_test",
-        "absl/base:sysinfo_test",
-        "absl/base:thread_identity_test",
-        "absl/base:tracing_internal_strong_test",
-        "absl/base:tracing_internal_weak_test",
         "absl/cleanup:cleanup_test",
         "absl/container:common_policy_traits_test",
         "absl/container:fixed_array_test",
@@ -277,21 +256,15 @@
         "absl/container:flat_hash_set_test",
         "absl/container:hash_function_defaults_test",
         "absl/container:inlined_vector_test",
-        "absl/container:node_hash_map_test",
-        "absl/container:node_hash_set_test",
         "absl/container:node_slot_policy_test",
         "absl/container:raw_hash_set_allocator_test",
-        "absl/container:raw_hash_set_resize_impl_test",
         "absl/container:sample_element_size_test",
         "absl/crc:crc32c_test",
         "absl/crc:crc_cord_state_test",
         "absl/crc:crc_memcpy_test",
         "absl/crc:non_temporal_memcpy_test",
-        "absl/debugging:bounded_utf8_length_sequence_test",
-        "absl/debugging:decode_rust_punycode_test",
         "absl/debugging:demangle_rust_test",
         "absl/debugging:stacktrace_test",
-        "absl/debugging:utf8_for_code_point_test",
         "absl/flags:flag_test",
         "absl/functional:any_invocable_test",
         "absl/functional:function_ref_test",
@@ -315,7 +288,6 @@
         "absl/log:vlog_is_on_test",
         "absl/log/internal:fnmatch_test",
         "absl/log/internal:stderr_log_sink_test",
-        "absl/log/internal:structured_proto_test",
         "absl/memory:memory_test",
         "absl/meta:type_traits_test",
         "absl/numeric:int128_test",
@@ -336,12 +308,9 @@
         "absl/strings:cord_rep_btree_reader_test",
         "absl/strings:cord_rep_btree_test",
         "absl/strings:cord_rep_crc_test",
-        "absl/strings:cord_test",
         "absl/strings:cordz_functions_test",
-        "absl/strings:cordz_handle_test",
         "absl/strings:cordz_info_statistics_test",
         "absl/strings:cordz_info_test",
-        "absl/strings:cordz_sample_token_test",
         "absl/strings:cordz_test",
         "absl/strings:cordz_update_scope_test",
         "absl/strings:cordz_update_tracker_test",
@@ -349,7 +318,6 @@
         "absl/strings:has_absl_stringify_test",
         "absl/strings:has_ostream_operator_test",
         "absl/strings:match_test",
-        "absl/strings:str_cat_test",
         "absl/strings:str_format_arg_test",
         "absl/strings:str_format_bind_test",
         "absl/strings:str_format_checker_test",
@@ -360,11 +328,7 @@
         "absl/strings:str_format_test",
         "absl/strings:str_replace_test",
         "absl/strings:string_view_test",
-        "absl/synchronization:barrier_test",
-        "absl/synchronization:graphcycles_test",
         "absl/synchronization:kernel_timeout_internal_test",
-        "absl/synchronization:mutex_test",
-        "absl/synchronization:per_thread_sem_test",
         "absl/synchronization:waiter_test",
         "absl/time:time_test",
         "absl/types:span_test",
--- third_party/abseil-cpp/absl/profiling/BUILD.gn	2025-10-24 18:42:30.000000000 +0200
+++ third_party/abseil-cpp/absl/profiling/BUILD.gn	2025-10-31 10:32:27.553808497 +0100
@@ -70,7 +70,6 @@
   deps = [
     "//third_party/abseil-cpp/absl/base",
     "//third_party/abseil-cpp/absl/base:config",
-    "//third_party/abseil-cpp/absl/base:raw_logging_internal",
     "//third_party/abseil-cpp/absl/container:btree",
     "//third_party/abseil-cpp/absl/container:flat_hash_map",
     "//third_party/abseil-cpp/absl/strings",
@@ -84,7 +83,6 @@
   deps = [
     ":profile_builder",
     "//third_party/abseil-cpp/absl/base:config",
-    "//third_party/abseil-cpp/absl/container:hashtablez_sampler",
     "//third_party/abseil-cpp/absl/status:statusor",
     "//third_party/abseil-cpp/absl/strings",
     "//third_party/abseil-cpp/absl/strings:string_view",
--- third_party/abseil-cpp/absl/random/internal/BUILD.gn	2025-10-24 18:42:30.000000000 +0200
+++ third_party/abseil-cpp/absl/random/internal/BUILD.gn	2025-10-31 10:32:27.614957300 +0100
@@ -18,7 +18,6 @@
   public = [ "distribution_caller.h" ]
   deps = [
     "//third_party/abseil-cpp/absl/base:config",
-    "//third_party/abseil-cpp/absl/base:fast_type_id",
     "//third_party/abseil-cpp/absl/meta:type_traits",
     "//third_party/abseil-cpp/absl/utility",
   ]
@@ -45,8 +44,6 @@
   deps = [
     ":fast_uniform_bits",
     "//third_party/abseil-cpp/absl/base:config",
-    "//third_party/abseil-cpp/absl/base:dynamic_annotations",
-    "//third_party/abseil-cpp/absl/base:raw_logging_internal",
     "//third_party/abseil-cpp/absl/strings",
     "//third_party/abseil-cpp/absl/strings:string_view",
     "//third_party/abseil-cpp/absl/types:optional",
@@ -65,7 +62,6 @@
     "//third_party/abseil-cpp/absl/base",
     "//third_party/abseil-cpp/absl/base:config",
     "//third_party/abseil-cpp/absl/base:core_headers",
-    "//third_party/abseil-cpp/absl/random:seed_gen_exception",
     "//third_party/abseil-cpp/absl/types:span",
   ]
   visibility = [ "//third_party/abseil-cpp/absl/random/*" ]
@@ -76,7 +72,6 @@
   public = [ "explicit_seed_seq.h" ]
   deps = [
     "//third_party/abseil-cpp/absl/base:config",
-    "//third_party/abseil-cpp/absl/base:endian",
   ]
   visibility = [ "//third_party/abseil-cpp/absl/random/*" ]
 }
@@ -169,7 +164,6 @@
   deps = [
     ":iostream_state_saver",
     ":randen",
-    "//third_party/abseil-cpp/absl/base:endian",
     "//third_party/abseil-cpp/absl/meta:type_traits",
   ]
   visibility = [ "//third_party/abseil-cpp/absl/random/*" ]
@@ -195,7 +189,6 @@
     ":platform",
     ":randen_hwaes",
     ":randen_slow",
-    "//third_party/abseil-cpp/absl/base:raw_logging_internal",
   ]
   visibility = [ "//third_party/abseil-cpp/absl/random/*" ]
 }
@@ -207,7 +200,6 @@
     ":platform",
     "//third_party/abseil-cpp/absl/base:config",
     "//third_party/abseil-cpp/absl/base:core_headers",
-    "//third_party/abseil-cpp/absl/base:endian",
     "//third_party/abseil-cpp/absl/numeric:int128",
   ]
   visibility = [ "//third_party/abseil-cpp/absl/random/*" ]
@@ -264,7 +256,6 @@
   deps = [
     "//third_party/abseil-cpp/absl/base:config",
     "//third_party/abseil-cpp/absl/base:core_headers",
-    "//third_party/abseil-cpp/absl/base:raw_logging_internal",
     "//third_party/abseil-cpp/absl/strings",
     "//third_party/abseil-cpp/absl/strings:str_format",
     "//third_party/abseil-cpp/absl/strings:string_view",
@@ -277,7 +268,6 @@
   public = [ "mock_helpers.h" ]
   deps = [
     "//third_party/abseil-cpp/absl/base:config",
-    "//third_party/abseil-cpp/absl/base:fast_type_id",
     "//third_party/abseil-cpp/absl/types:optional",
   ]
 }
@@ -288,7 +278,6 @@
   deps = [
     ":mock_helpers",
     "//third_party/abseil-cpp/absl/base:config",
-    "//third_party/abseil-cpp/absl/random:mocking_bit_gen",
     "//third_party/googletest:gtest",
   ]
 }
@@ -322,7 +311,6 @@
     ":iostream_state_saver",
     ":uniform_helper",
     "//third_party/abseil-cpp/absl/base:config",
-    "//third_party/abseil-cpp/absl/base:raw_logging_internal",
     "//third_party/abseil-cpp/absl/strings",
     "//third_party/abseil-cpp/absl/strings:string_view",
   ]
